app/models/early_career_payments/admin_tasks_presenter.rb

```diff
@@ -17,6 +17,20 @@ def identity_confirmation
       ]
     end
 
+    def qualifications
+      [].tap do |a|
+        a << [
+          "ITT start/end year",
+          I18n.t("answers.qts_award_years.on_date", year: eligibility.first_eligible_itt_academic_year.to_s(:long))
+        ]
+
+        a << [
+          "ITT subject",
+          eligibility.eligible_itt_subject.humanize
```

Comment:
@nomasprime have you applied `humanize` to all instances of `eligibility.eligible_itt_subject`? Or should it only be for specific instances? Looking at some of the specs they now seem inconsistent. 

e.g. https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/0bd4306f056e9dc773960f5b425e91bbac34e564/spec/features/early_career_payments_claim_spec.rb

Reply:
Not sure what you mean. Could you elaborate, maybe give line numbers?

Reply:
As discussed in zoom, what you have said in response makes sense. Happy to close